.travis.yml

```diff
@@ -125,7 +125,8 @@ install:
     # other dependencies.
 
 script:
-   - for mod in cadcutils; do cd $mod; python setup.py $SETUP_CMD || break -1; cd ..; done 
+   - cd cadcutils; python setup.py $SETUP_CMD || break -1; cd ..
```

Comment:
This is for the travis test runner to run code from multiple projects. Standard CADC to do it like this.